packages/slate-hyperscript/src/DecoratorPoint.js

```diff
@@ -0,0 +1,44 @@
+import { Range } from 'slate'
```

Comment:
Can you rename this file to not use uppercase characters? `decorator-point` instead.